DevOps/Readme.md

```diff
@@ -143,6 +143,13 @@
 - [Google Cloud Provider](https://cloud.google.com/docs)
 - [Azure](https://docs.microsoft.com/en-us/azure/?product=popular)
 
```

Comment:
Remove the white space. There is no need to separate video and written resources.

Reply:
I did the changed as you mentioned in the existing PR. If any changes need to be made, please let me know. Thank You.